src/components/InvoiceList/index.jsx

```diff
@@ -4,22 +4,39 @@ import InvoiceSummary from "../InvoiceSummary"
 export default function InvoiceList() {
 
     const [invoices, setInvoices] = useState([])
+    const [filterInvoices, setFilterInvoices] = useState(NaN)
+
     useEffect(() => {
-        fetch('https://invoicing-api.dev.io-academy.uk/invoices')
+        fetch(`https://invoicing-api.dev.io-academy.uk/invoices?status=${filterInvoices}`)
             .then(res => res.json())
             .then(invoiceData => {
                 setInvoices(invoiceData.data)
             }
             )
-    }, [])
+    }, [filterInvoices])
+
+    function filterStatus(e) {
+        setFilterInvoices(Number(e.target.value))
+    }
 
     return (
-        <div className="flex flex-col items-center px-16 bg-slate-50">
-            {invoices.map(invoice => {
-                return (
-                    <InvoiceSummary key={invoice.id} invoiceID={invoice.invoice_id} invoiceTotal={invoice.invoice_total} invoiceName={invoice.name} invoiceStat={invoice.status_name} invoiceDue={invoice.due} />
-                )
-            })}
+        <div className="flex flex-col items-center bg-slate-50">
+            <div className="flex mb-5">
+                <select onChange={filterStatus} className="bg-slate-50 text-blue-950 text-sm font-bold">
+                    <option value="NaN">Filter by Status</option>
+                    <option value="1">Filter by Paid</option>
+                    <option value="2">Filter by Pending</option>
+                    <option value="3">Filter by Cancelled</option>
```

Comment:
Can you update this to just be Paid, Pending and Cencelled - but keep the filter by on the Filter by Status option